testing/tests/ui/filter-recursion.stderr

```diff
@@ -1,6 +1,6 @@
-error: unclosed expression, missing "}}"
-       failed to parse template source at row 1, column 0 near:
-       "{{ s|a|a|a|a|a|a|a|A|a|a|a|a|a|a|a|a|a|a"...
+error: failed to parse template source
```

Comment:
Thinking out loud but the error is not great. It should indicate that it hits the recursion limit. Gonna open an issue for that.